hail_search/constants.py

```diff
@@ -0,0 +1,13 @@
+GENOME_VERSION_GRCh38_DISPLAY = 'GRCh38'
```

Comment:
Is it possible to share the same constants with the ES search? Same for the rest of the constants?